tests/test_routes.py

```diff
@@ -5,13 +5,11 @@
   nosetests -v --with-spec --spec-color
   coverage report -m
 """
-import os
 import logging
 from datetime import timedelta
 from unittest import TestCase
-from unittest.mock import MagicMock, patch
 from service import app
-from service.models import Promotion, DataValidationError, db
```

Comment:
we need import db, I used db. This request was based on previous codes.

Reply:
updated!